backend/pharmacy/api/urls.py

```diff
@@ -1,17 +1,34 @@
+from django.conf import settings
 from django.urls import path
 from . import views
 from rest_framework_simplejwt.views import (
     TokenRefreshView,
     TokenVerifyView
 )
 from .views import MyTokenObtainPairView
+from rest_framework.urlpatterns import format_suffix_patterns
+
+from django.conf.urls.static import static
+from django.contrib.staticfiles.urls import staticfiles_urlpatterns
+
 
 urlpatterns = [
-    path('', views.getData),
-    path('post/', views.postData),
-    path('update/', views.updateData),
-    path('get/', views.getRoutes),
+    path('', views.DataList.as_view(), name='get'),
+    path('<int:pk>/', views.Data.as_view()),
+
+    # User
+    path('user/', views.User.as_view()),
+
     path('token/', MyTokenObtainPairView.as_view(), name='token_obtain_pair'),
     path('token/refresh/', TokenRefreshView.as_view(), name='token_refresh'),
     path('token/verify/', TokenVerifyView.as_view(), name='token_verify'),
 ]
+
+urlpatterns = format_suffix_patterns(urlpatterns)
+
+
+# For media files   
```

Comment:
This section is telling my application: hey check settings and look for `MEDIA_URL` variable and set the document root to that... this will help django app to know that where the hell is my media files. i.e. images

Reply:
can we say that this section is particularly handling images